pkg/apis/kops/validation/validation.go

```diff
@@ -517,6 +517,12 @@ func validateKubelet(k *kops.KubeletConfigSpec, c *kops.Cluster, kubeletPath *fi
 			}
 		}
 
+		if k.EnableCadvisorJsonEndpoints != nil {
+			if c.IsKubernetesGTE("1.18") && c.IsKubernetesLT("1.21") {
+				allErrs = append(allErrs, field.Forbidden(kubeletPath.Child("enableCadvisorJsonEndpoints"), "enableCadvisorJsonEndpoints requires at least Kubernetes 1.18"))
```

Comment:
Sorry, my mistake. Please change and squash one more time.
```suggestion
			if c.IsKubernetesLT("1.18") && c.IsKubernetesGTE("1.21") {
				allErrs = append(allErrs, field.Forbidden(kubeletPath.Child("enableCadvisorJsonEndpoints"), "enableCadvisorJsonEndpoints requires at Kubernetes 1.18-1.20"))
```
```suggestion
			if c.IsKubernetesGTE("1.18") && c.IsKubernetesLT("1.21") {
				allErrs = append(allErrs, field.Forbidden(kubeletPath.Child("enableCadvisorJsonEndpoints"), "enableCadvisorJsonEndpoints requires at least Kubernetes 1.18"))
```

Reply:
> Sorry, my mistake. Please change and squash one more time.

No worries, I should have caught that.

Fixed